index.html

```diff
@@ -119,19 +121,19 @@ <h3 class="worktitle">My Recent Work</h3>
                 </p>
                 <ul class="usedcode">
                     <li>css</li>
-                    <li><img src="./pictures/Rectangle70.png" alt=""></li>
+                    <li><img src="./pictures/Rectangle70.png" alt="separator line between technologies used"></li>
                     <li>html</li>
-                    <li><img src="./pictures/Rectangle70.png" alt=""></li>
+                    <li><img src="./pictures/Rectangle70.png" alt="separator line between technologies used"></li>
                     <li>bootstrap</li>
-                    <li><img src="./pictures/Rectangle70.png" alt=""></li>
+                    <li><img src="./pictures/Rectangle70.png" alt="separator line between technologies used"></li>
```

Comment:
- [ ] I think you should consider modifying the alt attribute given for the separator lines images because the images are not important for understanding the content of the page as it just serves as decoration. Please set the alt to null (alt="") 👍 

I think If the images were removed, the user will still get all the information from the page 😄.